tests/dist/chats.spec.dev.js

```diff
@@ -95,35 +100,35 @@ describe('chat resolvers', function () {
 
             _context2.next = 13;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n        mutation{\n            createDirectChat(data: {\n              organizationId: "'
-                  .concat(createdOrgId, '"\n              userIds: ["')
-                  .concat(loggedInUserId, '", "')
-                  .concat(
-                    newUserId,
-                    '"]\n            }){\n              _id\n            }\n          }\n                '
-                  ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n        mutation{\n            createDirectChat(data: {\n              organizationId: "'
+                    .concat(createdOrgId, '"\n              userIds: ["')
+                    .concat(loggedInUserId, '", "')
+                    .concat(
+                      newUserId,
+                      '"]\n            }){\n              _id\n            }\n          }\n                '
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.